NOTICE.md

```diff
@@ -0,0 +1,89 @@
+# Notices for Eclipse Hara
+
+This content is produced and maintained by the Eclipse Hara project.
+
+    * Project home: https://projects.eclipse.org/projects/iot.hawkbit.hara
+
+## Trademarks
+
+Eclipse Hara are trademarks of the Eclipse Foundation. Eclipse, and 
```

Comment:
fix: "Eclipse Hara **are** trademarks of the Eclipse Foundation"

"Eclipse Hara is a trademark of the Eclipse Foundation."